### PR TITLE
Enable rpm signature checking when building images

### PR DESCRIPTION
**Pull request by bbguimaraes**
@jupierce: PTAL.

### CODE REVIEW

**Comment by bbguimaraes** [comment]
@jupierce: updated, PTAL.

File: build-scripts/ose_images/ose_images.sh [hunk 1]
```diff
@@ -515,12 +515,12 @@ check_build_dependencies() {
...
+    #rhpkg container-build --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-unsigned-latest.repo >> ${workingdir}/logs/${container}.buildlog 2>&1 &
+    #rhpkg container-build --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-unsigned-errata-building.repo >> ${workingdir}/logs/${container}.buildlog 2>&1 &
+    #rhpkg container-build --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-unsigned-errata-latest.repo >> ${workingdir}/logs/${container}.buildlog 2>&1 &
+    #rhpkg container-build --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-signed-building.repo >> ${workingdir}/logs/${container}.buildlog 2>&1 &
+    #rhpkg container-build --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-signed-latest.repo >> ${workingdir}/logs/${container}.buildlog 2>&1 &
```
Comment (bbguimaraes) [inline build-scripts/ose_images/ose_images.sh]: Are these commented lines still relevant?

Comment (jupierce) [reply build-scripts/ose_images/ose_images.sh]: Not relevant any longer, but perfectly fine to have changed.

**Comment by jupierce** [comment]
@smunilla FYI. refresh-images will now have an option which attempts to verify RPM signatures. Try using the aos-signed-building-betatest.repo variant next time you run the job.

**Pull request merged**